lib/gems-cli/paginator.rb

```diff
@@ -4,8 +4,8 @@ class Paginator
   attr_reader :pages, :total_entries
   attr_accessor :ary, :per_page, :page, :paged_results
 
-  def initialize(args={})
-    @ary = args[:ary] || Array.new
+  def initialize args={}
```

Comment:
Use def with parentheses when there are parameters.<br>Surrounding space missing in default value assignment.